custom_model_runner/datarobot_drum/resource/drum_server_helpers.py

```diff
@@ -0,0 +1,203 @@
+import os
```

Comment:
obvoiusly redeclaring a bunch of functionality from tests inside of drum is kind of silly- any reason not to just move these into drum and call from that location in tests? alternatively, maybe there's a better way to call drum server from within a drum fit run and I should be taking a different approach altogether?

Reply:
@yakov-g @4dahalibut  is there a preferred way to call drum server from `run_test_predict`? if these helpers can live in drum I'll just move them over fully, but very open to other approaches